### ISSUE TITLE
static IP not set correctly

### ISSUE THREAD

**Issue by geoHeil** [cogitatio/vagrant-hostsupdater#132]
When using `Vagrant 1.9.1` with the following file:
```

Vagrant.configure("2") do |config|
  config.vm.box = "centos/7"
  config.vm.network :private_network, ip: "192.168.3.10"
  config.vm.hostname = "testing.vagrant"
end
```

this is the output for ifconfig is:
```
ifconfig
eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 172.16.75.137  netmask 255.255.255.0  broadcast 172.16.75.255
        inet6 fe80::20c:29ff:feea:279e  prefixlen 64  scopeid 0x20<link>
        ether 00:0c:29:ea:27:9e  txqueuelen 1000  (Ethernet)
        RX packets 7256  bytes 9240928 (8.8 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 3524  bytes 284441 (277.7 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
```

as you can see: `inet 172.16.75.137` is not set to `192.168.3.10` what is wrong here?

**Comment by cbirajdar** [comment]
@geoHeil That seems to be an issue with vagrant, not the hostsupdater plugin. Please see https://github.com/mitchellh/vagrant/issues/8115 for more details. Per comments, bug fix for this issue is slated for 1.9.2.

For now, try one of these:
- Downgrade to 1.9.0
- Add ```config.vm.provision "shell", inline: "sudo systemctl restart network.service"```

Hope that helps!

**Comment by cgsmith** [comment]
Thanks for helping with this @cbirajdar 👍
